google/cloud/storage/tests/testbench.py

```diff
@@ -87,7 +87,10 @@ def __init__(self, gcs_url, bucket_name, name, generation, request):
         self.object_id = bucket_name + '/o/' + name + '/' + str(generation)
         now = time.gmtime(time.time())
         timestamp = time.strftime('%Y-%m-%dT%H:%M:%SZ', now)
-        self.media = request.data
+        if request.environ.get('HTTP_TRANSFER_ENCODING', '') == 'chunked':
+            self.media = ''.join(request.environ.get('wsgi.input').readlines())
```

Comment:
Why `''.join(foo.readlines())`?  I believe `foo.read()` should return the entire contents.

Reply:
Fixed.  For some reason I tried that and did not work, but I was trying many things so I probably had something else wrong.